### ISSUE TITLE
Severerely outdated documentation and inability to merge original docs

### ISSUE THREAD

**Issue by viniciusdeliz** [vuejs-translations/docs-pt#85]
There were many changes in the official documentation repository since this branch was created. The documentation is alive, as expected, and always requires updates.

However the way this current repository was created does not accept changes from the original docs. We cannot seamlessly merge and update the content from its source of truth.

As I mentioned in [another issue](https://github.com/nazarepiedady/vue3-docs-pt/issues/33#issuecomment-1605087351), I had to rebase all the documentation, checking every update and difference from the original documentation against this current repository, so I could be able to easily ingest and keep the original [vuejs/docs](https://github.com/vuejs/docs) as my source of truth.

I understand that it might be hard to check for differences (especially with the way this branch is right now, since we can't compare it directly with the original docs), but there are many. Just to mention some: several playground links were changed (they went from `http://sfc.vue...` to `http://play.vue...`), many identifier links were added to files (the # after the title), content was removed and introduced in different files, package.json is way different, and many more. Evan You and many other core members are [changing the documentation constantly](https://github.com/vuejs/docs/commits/main), and we are dedicating effort for translating things that are now replaced or structured differently.

I'm worried that we are working on a repository that is not following the basic steps from the [vuejs-translations guidelines](https://github.com/vuejs-translations), and that does not have the ability to keep its content up to date. For me this issue is a severe blocker that should be addressed immediately.

**Comment by nazarepiedady** [comment]
@viniciusdeliz it seems that you do not have read the documentation markdown file.

All changes you are pointing out, and I have applied them in each commit I have done.
And I still being updating the files little by little, I can see you a little distracted because
this documentation is using `https://play.vue` for a long time.

So there is not any severely outdated documentation.
As I said before the other translations are not forks of the [vuejs/docs](https://github.com/vuejs/docs).

I read the [vuejs-translations/guidelines](https://github.com/vuejs-translations/guidelines) before starting the translation,
and the project is still following the recommendations.

I suggest you update your copy of this repository in case you are outdated.

**Comment by viniciusdeliz** [comment]
@nazarepiedady 

Do not want to point that specifically, but yes, there are many translations that are forks of vuejs/docs.
If they're not, that's okay, but they probably need to keep a commit history similar to the original vuejs/docs. 

Let's take the japanese version as an example: https://github.com/vuejs-translations/docs-ja

1. Clone docs-ja into your local environment
2. Add [vuejs/docs](https://github.com/vuejs/docs) as a remote origin
3. Merge `main` of vuejs/docs into docs-ja

You'll see many merge conflicts. That's a good thing! It means that we are able to keep the docs-ja repository plainly synched with the original vuejs/docs.

Now let's try that with vue3-docs-pt:

<img width="502" alt="fatal-unrelated-histories" src="https://github.com/nazarepiedady/vue3-docs-pt/assets/5748616/fbd8c53c-02d5-46c6-9047-2ac905298ea5">

Git refuses to merge the original vuejs/docs into the current repository because the commit history is completely different.

Updating files little by little is impairing the progress of the translation, therefore delaying it to be published and to be served for Portuguese speakers. I stand for the need to keep a workflow that easily ingest changes from the upstream.

But as I mentioned [here](https://github.com/nazarepiedady/vue3-docs-pt/issues/84#issuecomment-1614685272), if all files are already in Portuguese, I suggest that you mark the translation as ready and start the process of migrating and publishing it to the official vuejs-translations repository.

**Comment by nazarepiedady** [comment]
@viniciusdeliz I contacted the colleagues from [vuejs-translations](https://github.com/vuejs-translations) in order to add the [vue3-docs-pt](https://github.comnazarepiedady/vue3-docs-pt) to the organisation. I configured the [ryu-cho](https://github.com/vuejs-translations/docs-ja/blob/main/.github/workflows/ryu-cho.yaml) that uses an approach that I used in [vitejs/docs-pt](https://github.com/vitejs/docs-pt).

**Comment by viniciusdeliz** [comment]
I still think we should be able to ingest changes effortlessly from the original docs main branch using `git merge` (docs source and translated repository should have related commit histories), but I'll close this story anyway so we can open the road for revision and proofreading

**Comment by nazarepiedady** [comment]
@viniciusdeliz At the time I did what was recommended by the Vue and Vite core programmers because this way the repository would have its git history but the changes on the official English version do not drastically add updates so it is still easy to update the content.
